.github/workflows/triage-move-labelled.yml

```diff
@@ -47,45 +47,30 @@ jobs:
           PROJECT_ID: "PN_kwDOAM0swc0sUA"
           GITHUB_TOKEN: ${{ secrets.ELEMENT_BOT_TOKEN }}
 
-  spaces_issues_to_old_board:
-    name: Spaces issues to old Delight project board
-    runs-on: ubuntu-latest
-    if: >
-        contains(github.event.issue.labels.*.name, 'A-Spaces') ||
-        contains(github.event.issue.labels.*.name, 'A-Space-Settings') ||
-        contains(github.event.issue.labels.*.name, 'A-Subspaces')
-    steps:
-      - uses: konradpabjan/move-labeled-or-milestoned-issue@219d384e03fa4b6460cd24f9f37d19eb033a4338
-        with:
-          action-token: "${{ secrets.ELEMENT_BOT_TOKEN }}"
-          project-url: "https://github.com/orgs/vector-im/projects/6"
-          column-name: "📥 Inbox"
-          label-name: "A-Spaces"
-
-  spaces_issues_to_new_board:
-    name: Spaces issues to new Delight project board
-    runs-on: ubuntu-latest
-    if: >
-        contains(github.event.issue.labels.*.name, 'A-Spaces') ||
-        contains(github.event.issue.labels.*.name, 'A-Space-Settings') ||
-        contains(github.event.issue.labels.*.name, 'A-Subspaces')
-    steps:
-      - uses: octokit/graphql-action@v2.x
-        with:
-          headers: '{"GraphQL-Features": "projects_next_graphql"}'
-          query: |
-            mutation add_to_project($projectid:String!,$contentid:String!) {
-              addProjectNextItem(input:{projectId:$projectid contentId:$contentid}) {
-                projectNextItem {
-                  id
-                }
-              }
-            }
-          projectid: ${{ env.PROJECT_ID }}
-          contentid: ${{ github.event.issue.node_id }}
-        env:
-          PROJECT_ID: "PN_kwDOAM0swc1HvQ"
-          GITHUB_TOKEN: ${{ secrets.ELEMENT_BOT_TOKEN }}
+#  delight_issues_to_board:
```

Comment:
for my understanding, are we planning to enable this in the future?

Reply:
It depends on what happens with hiring, but I am assuming we will have an Android developer on the Delight team at some point in the future, even if it's to work on something other than Spaces.

I also want to be ready for people other than me to be able to make at least simple changes to the board.